informative-indexer/flusher/block_results.go

```diff
@@ -24,7 +24,7 @@ func (f *Flusher) parseAndInsertTransactionEvents(parentCtx context.Context, dbT
 	f.dbBatchInsert = NewDBBatchInsert()
 	txs := make([]*db.Transaction, 0)
 	txEvents := make([]*db.TransactionEvent, 0)
-	for i, txResult := range blockResults.Txs {
+	for i, txResult := range *blockResults.Txs {
```

Comment:
what do you think we check *blockResults.Txs is not nil i know that i wont happen, but in most real-world cases, adding a simple nil check is worth the extra safety and future-proofing. im 50/50 with this